### ISSUE TITLE
Use a newer istio version (e.g., 1.5.4)

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#1825]
Keptn 0.6.2 deploys Istio 1.3.x. We should upgrade to a newer istio version (e.g., 1.5.4) to also easily support tutorials like this one: https://medium.com/keptn/securing-your-kubernetes-services-with-tls-certificates-issues-by-lets-encrypt-how-to-setup-5097954842de

Also, noteworthy: Support for 1.3 has ended - see https://istio.io/news/support/announcing-1.3-eol-final/
Support for 1.4 is only there until June: https://istio.io/news/support/announcing-1.4-eol/

**Comment by agrimmer** [comment]
Related to https://github.com/keptn/keptn/issues/1406

**Comment by christian-kreuzberger-dtx** [comment]
In my opinion the first step for the user is to update istio manually (using the official istio documentation). We should point the user to the documentation (it might even be several steps, e.g., upgrading from 1.3 to 1.4, then from 1.4 to 1.5, etc...).

The upgrader should really not perform the istio upgrade behind the scenes. While this might be a convenient option for demos and workshops, I am pretty sure that administrators will not want to see their Istio Installation changed by a third party program.

> How can we ensure that Istio can safely be updated (e.g. is not used for production traffic)?

Short version: we *can`t*. We need to document that the upgrade will result in downtime because that's how the Istio upgrade works.

**Comment by christian-kreuzberger-dtx** [comment]
This issue will become obsolete once we implement [KEP 18](https://github.com/keptn/enhancement-proposals/pull/18).

**Comment by christian-kreuzberger-dtx** [comment]
Closing this as it is implemented in KEP 18.
